make a change

danvk opened the pull request:
I hope you like my changes!

danvk-test commented:
Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/danvk/test-reviewable/1)

Reviewed files:
- LICENSE @ r1
- RangeHTTPServer.py @ r1

---

I see a few problems with this change!

---

<sup>**[LICENSE, line 9 [r1]](https://reviewable.io:443/reviews/danvk/test-reviewable/1#-Jj0p90elWPtTs6jDVFD)** ([raw file](https://github.com/danvk/test-reviewable/blob/7a0f2ca176fdf683035bcbbc486da8bc2ad0345b/LICENSE#L9)):</sup>
This seems risky!

---

<sup>**[RangeHTTPServer.py, line 50 [r1]](https://reviewable.io:443/reviews/danvk/test-reviewable/1#-Jj0pUhWX2a_DV_5FV3L)** ([raw file](https://github.com/danvk/test-reviewable/blob/7a0f2ca176fdf683035bcbbc486da8bc2ad0345b/RangeHTTPServer.py#L50)):</sup>
Did you forget to run the tests?

---

<sup>**[RangeHTTPServer.py, line 56 [r1]](https://reviewable.io:443/reviews/danvk/test-reviewable/1#-Jj0pY6quZukdlFwnkEZ)** ([raw file](https://github.com/danvk/test-reviewable/blob/7a0f2ca176fdf683035bcbbc486da8bc2ad0345b/RangeHTTPServer.py#L56)):</sup>
This comment seems inappropriate given what the function does.

---

<!-- Sent from Reviewable.io -->

danvk commented:
Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/danvk/test-reviewable/1)

---

I've addressed all comments, PTAL.

---

<sup>**[LICENSE, line 9 [r1]](https://reviewable.io:443/reviews/danvk/test-reviewable/1#-Jj0p90elWPtTs6jDVFD)** ([raw file](https://github.com/danvk/test-reviewable/blob/7a0f2ca176fdf683035bcbbc486da8bc2ad0345b/LICENSE#L9)):</sup>
Done.

---

<sup>**[RangeHTTPServer.py, line 50 [r1]](https://reviewable.io:443/reviews/danvk/test-reviewable/1#-Jj0pUhWX2a_DV_5FV3L)** ([raw file](https://github.com/danvk/test-reviewable/blob/7a0f2ca176fdf683035bcbbc486da8bc2ad0345b/RangeHTTPServer.py#L50)):</sup>
I sure did! Reverted.

---

<sup>**[RangeHTTPServer.py, line 56 [r1]](https://reviewable.io:443/reviews/danvk/test-reviewable/1#-Jj0pY6quZukdlFwnkEZ)** ([raw file](https://github.com/danvk/test-reviewable/blob/7a0f2ca176fdf683035bcbbc486da8bc2ad0345b/RangeHTTPServer.py#L56)):</sup>
You're right, I've updated the docstring and renamed the function.

---

<!-- Sent from Reviewable.io -->
